src/components/custom-tree-item.js

```diff
@@ -44,39 +45,38 @@ const CustomContent = React.forwardRef(function CustomContent(props, ref) {
 
     return (
         // eslint-disable-next-line jsx-a11y/no-static-element-interactions
-        <div
-            className={clsx(className, classes.root, {
-                [classes.expanded]: expanded,
-                [classes.selected]: selected,
-                [classes.focused]: focused,
-                [classes.disabled]: disabled,
-            })}
+        <Box
+            sx={mergeSx(
+                className,
```

Comment:
Why you put classname into sx ? It's not even used anymore, or I missed it

Reply:
moved to className prop